MAiD - Case Management App/force-app/main/default/permissionsets/MAiD_System_Administrator.permissionset-meta.xml

```diff
@@ -4192,15 +4207,6 @@
         <object>ScorecardMetric</object>
         <viewAllRecords>true</viewAllRecords>
     </objectPermissions>
-    <objectPermissions>
```

Comment:
@AmishMistri Why are we removing this object permissions? Can you make sure the file matches the old commit or dev branch?